packages/client/src/ActionsAndReducers/dbWargames/wargames_ActionCreators.js

```diff
@@ -146,6 +151,13 @@ export const populateWargameStore = () => {
   }
 }
 
+export const pingServerWithInterval = () => async (dispatch) => {
+  setInterval(async () => {
+    const serverStatus = await wargamesApi.pingServer()
+    dispatch(saveServerStatus(serverStatus))
+  }, 10000)
```

Comment:
Could you store the 10000 as a constant in the global consts file please?

Reply:
sure